core/new-gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -360,12 +361,38 @@ export class JointGraphWrapper {
    */
   public highlightOperators(...operatorIDs: string[]): void {
     const highlightedOperatorIDs: string[] = [];
-    operatorIDs.forEach(operatorID =>
-      this.highlightElement(operatorID, this.currentHighlightedOperators, highlightedOperatorIDs)
-    );
+    const highlightedLinkIDs: string[] = [];
+    operatorIDs.forEach(operatorID => {
+      this.highlightElement(operatorID, this.currentHighlightedOperators, highlightedOperatorIDs);
+
+      // get all the links whose both ends are connected to the currently highlighted operators
+      const allLinks = this.workflowActionService.getTexeraGraph().getAllLinks();
```

Comment:
Thanks for the advice! I have implemented the "auto-selecting links between two highlighted operators" part in the workflow editor component instead of the joint graph wrapper.